src/Models/SiteTreeLink.php

```diff
@@ -134,4 +134,19 @@ public function getTitle(): ?string
         // Use page title as a default value in case CMS user didn't provide the title
         return $page->Title;
     }
+
+    public function getLinkTypeDefaultTitle(): string
+    {
+        $pageExist = $this->Page()->exists() && $this->getURL();
+
+        // If the page doesn't exist, we can't get the title
+        if (!$pageExist) {
+            return _t(
+                'LinkField.DEFAULT_LINK_TITLE_WRONG_SITE_TREE_LINK',
+                'Page missing',
+            );
+        }
+
+        return $this->getTitle();
```

Comment:
```suggestion
        return $page->Title;
```
The page title is the default.

Reply:
Updated